index.php

```diff
@@ -247,3 +316,7 @@ function the_gutenberg_project() {
 	</div>
 	<?php
 }
+
+register_block( 'core/html', function( $attributes ) {
```

Comment:
Seems temporary, but just for clarity, the anonymous function callable is only available in PHP versions 5.3.0 and newer ([reference](http://php.net/manual/en/functions.anonymous.php)). WordPress supports as far back as 5.2.4.